assets/js/state/sagas/checksSelection.js

```diff
@@ -3,7 +3,7 @@ import { post } from '@lib/network';
 
 import { TARGET_HOST, TARGET_CLUSTER } from '@lib/model';
 
-import { notify } from '@state/actions/notifications';
+import { notify } from '@state/sagas/notifications';
```

Comment:
`notify` is an actions, so it should still come from the reducer new file, don't you think?

Reply:
Yeah!